doc/topics/cloud/profitbricks.rst

```diff
@@ -228,5 +259,5 @@ wait_for_timeout
     The timeout to wait in seconds for provisioning resources such as servers.
     The default wait_for_timeout is 15 minutes.
 
-For more information concerning cloud profiles, see :ref:`here
-<salt-cloud-profiles>`.
+For more information concerning cloud profiles, see :doc:`here
```

Comment:
Is there a reason you changed this reference from using `ref` to `doc`? We need to use `ref` here as using `doc` should no longer be done. This is failing the following test: https://jenkins.saltstack.com/job/PR/job/salt-pr-linode-ubuntu14-n/18095/testReport/junit/unit.test_doc/DocTestCase/test_check_for_doc_inline_markup/.